stdlib/SparseArrays/test/sparse.jl

```diff
@@ -1048,110 +1048,112 @@ end
     @test iA === iS === nothing
 end
 
-# findmin/findmax/minumum/maximum
+@testset "findmin/findmax/minumum/maximum" begin
```

Comment:
This just adds a testset to fix:
```
Deprecated syntax `implicit assignment to global variable `A``
```
 see https://github.com/JuliaLang/julia/pull/25249/files?w=1#diff-6b5c2f417a1aa03f6bfdfb283f4bd01c

Reply:
Good catch!